pennylane_lightning/core/src/utils/NDPermuter.hpp

```diff
@@ -68,7 +68,8 @@ template <class PermuterBackend> class Permuter {
      */
     template <class T>
     void Transpose(const std::vector<T> &data_in,
-                   const std::vector<size_t> &shape, std::vector<T> &data_out,
```

Comment:
Like that too. I think we should even make a `size_t` => `std::size_t` PR and be rid of it.